client/src/app/site/motions/modules/motion-import/motion-import-list.component.html

```diff
@@ -25,7 +25,7 @@ <h2>{{ 'Import motions' | translate }}</h2>
     <ul>
         <li>
             {{
-                'Number, reason, submitter, category, origin and motion block are optional and may be empty.'
+                'Identifier, reason, submitter, category, origin and motion block are optional and may be empty.'
```

Comment:
Number is correct - identifier is the OS3 term

Reply:
Ah, okay :+1: